src/reference/asciidoc/amqp.adoc

```diff
@@ -1703,8 +1704,38 @@ public String spelReplyTo() {
 
 The expression must evaluate to a `String`, which can be a simple queue name (sent to the default exchange) or with
 the form `exchange/routingKey` as discussed above.
-The expression is evaluated once, during context initialization.
-For dynamic reply routing, the message sender should include a `reply_to` message property.
+
+NOTE: The `#{...}` expression is evaluated once, during initialization.
+
+For dynamic reply routing, the message sender should include a `reply_to` message property or use the alternate
+runtime SpEL expression described below.
+
+Starting with _version 1.6_, the `@SendTo` can be a SpEL expression that is evaluated at runtime against the request
+and reply:
+
+[source, java]
+----
+@RabbitListener(queues = "test.sendTo.spel")
+@SendTo("!{'some.reply.queue.with.' + result.queueName}")
+public Bar capitalizeWithSendToSpel(Foo foo) {
+    return processTheFooAndReturnABar(foo);
+}
+----
+
+The runtime nature of the SpEL expression is indicated with the `!{...}` prefix.
```

Comment:
This isn't `prefix` anymore.
Maybe `delimiters` like it is in the Core SpEL doc:
> Each evaluation block is delimited with prefix and suffix characters that you can define, a common choice is to use `#{ }` as the delimiters

Let me know your thought and I'll fix on merge.